### PR TITLE
Pt 164657547 avoid contract account creation

### PR DESCRIPTION
**Pull request by gorillainduction**
https://www.pivotaltracker.com/story/show/164657547

### CODE REVIEW

File: apps/aecontract/test/aecontract_SUITE.erl [hunk 1]
```diff
@@ -1132,10 +1173,9 @@ sophia_identity(_Cfg) ->
...
     state(aect_test_utils:new_state()),
-    ForkHeight    = ?config(fork_height, Cfg),
-    RomaHeight    = ForkHeight - 1,
-    MinervaHeight = ForkHeight + 1,
-    MinervaGasPrice = aec_governance:minimum_gas_price(MinervaHeight),
+    ForkHeights   = ?config(fork_heights, Cfg),
+    RomaHeight    = maps:get(minerva, ForkHeights) - 1,
+    MinervaHeight = maps:get(fortuna, ForkHeights) - 1,
     MinerMinGasPrice= aec_tx_pool:minimum_miner_gas_price(),
     MinGasPrice   = max(MinerMinGasPrice, MinerMinGasPrice),
```
Comment (ThomasArts) [inline apps/aecontract/test/aecontract_SUITE.erl]: This is weird, getting maximum of two equal numbers.

Comment (gorillainduction) [reply apps/aecontract/test/aecontract_SUITE.erl]: This should probably be the unused `MinervaGasPrice` that I removed.

Comment (gorillainduction) [reply apps/aecontract/test/aecontract_SUITE.erl]: I can fix this

File: apps/aecontract/test/aecontract_SUITE.erl [hunk 2]
```diff
@@ -434,6 +441,40 @@ create_contract_init_error(_Cfg) ->
...
+    {ok, S2Minerva} = sign_and_apply_transaction(Tx, PrivKey, S1, MinervaHeight),
+    {ok, S2Fortuna} = sign_and_apply_transaction(Tx, PrivKey, S1, FortunaHeight),
+
+    %% Check that the contract is not created
+    ContractKey = aect_contracts:compute_contract_pubkey(PubKey, aetx:nonce(Tx)),
+    ?assertMatch({none, _}, lookup_contract_by_id(ContractKey, S2Minerva)),
+    ?assertMatch({none, _}, lookup_contract_by_id(ContractKey, S2Fortuna)),
+
+    %% In Minerva, the failing init should     create the contract account.
+    %% In Fortuna, the failing init should NOT create the contract account.
```
Comment (lucafavatella) [inline apps/aecontract/test/aecontract_SUITE.erl]: I see this is about the contract account - not the contract - so it is not caught by test case `create_contract_init_error`. Ok.

**Comment by gorillainduction** [comment]
Rebased and squashed to resolve conflict.

**Pull request merged**